common/testing/virtual-threads/src/main/java/module-info.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Virtual Thread testing features for Helidon test extensions.
+ */
+module io.helidon.common.testing.vitualthreads {
+    requires jdk.jfr;
+    exports io.helidon.common.testing.virtualthreads to
```

Comment:
I think we do not need to restrict this. Or do we want to keep it as internal until we know the API is settled?

Reply:
That was my reasoning, rather to wait for some major release with introducing a new API